crates/ingress-kafka/Cargo.toml

```diff
@@ -25,7 +25,7 @@ derive_builder = { workspace = true }
 drain = { workspace = true }
 opentelemetry_api = { workspace = true }
 prost = { workspace = true }
-rdkafka = { version = "0.34", features = ["cmake-build"] }
+rdkafka = { version = "0.34", features = ["libz-static"] }
```

Comment:
Is this needed for cross? If we are no longer using cross, should we revert this change?

Reply:
the cmake build for librdkafka is experimental and i found it highly unreliable on all targets when cross compiling. we still need cross on linux, and to cross compile on mac using clang also. ive synced with francesco and the cmake build isnt needed. and libz-static was being set implicitly in most builds anyway, just wanted to set it explicitly - we don't ever want to link against local libz

Reply:
Thanks for the clarification. Wondering why this has never been a problem on my machine also when cross compiling.

Reply:
the problem with the cmake build was that it looks around in your build environment to decide what to use and was making bad choices. this may be fine on your machine but i was finding it to work inconsistently in cross environments - we want the build process to be as predictable as possible, and here that seems to be the non cmake process